magento/pwa-studio
# #2804 :- Add "maskable icon" to Venia for Google Lighthouse [PWA]

## Description

This PR will add the maskable icon

## Related Issue





Closes https://github.com/magento/pwa-studio/issues/2804

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. On an Android device, go to develop.pwa-venia.com on chrome and in the top right menu select "add to home page".
2. Now go to [this PR](pr-2818.pwa-venia.com) and do the same.
3. Compare the icons on the home page, they should look identical.
4. Run lighthouse against this PR, it should not have the "maskable icon" warning.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
